src/bitcoin-cli.cpp

```diff
@@ -491,11 +491,16 @@ class NetinfoRequestHandler : public BaseRequestHandler
         result += strprintf("out    %5i   %5i   %5i   %5i   %5i\n", ipv4_o, ipv6_o, onion_o, total_o, block_relay_o);
         result += strprintf("total  %5i   %5i   %5i   %5i   %5i\n", ipv4_i + ipv4_o, ipv6_i + ipv6_o, onion_i + onion_o, total_i + total_o, block_relay_i + block_relay_o);
 
-        // Report local addresses, ports, and scores.
-        result += "\nLocal addresses";
+        // Report reachable networks and local addresses info.
+        result += "\nnetworks:";
+        for (const UniValue& network : networkinfo["networks"].getValues()) {
+            const std::string name{network["name"].get_str()};
+            if (!name.empty() && network["reachable"].get_bool()) result += " " + name;
+        }
```

Comment:
This produces the following output with `bitcoin-cli -testnet -netinfo` for me

```
Bitcoin Core v0.20.99.0-8b0aa43e2 testnet - 70016/Satoshi:0.20.99/

        ipv4    ipv6   onion   total  block-relay
in         0       0       0       0       0
out        9       0       1      10       2
total      9       0       1      10       2

networks: ipv4 ipv6 onion - local addresses:
2002:b10c:b10c:0:1234::5057     port  18333    score      1
2002:b10c:b10c:0:1234::33ed     port  18333    score      1
b10cabcdefaqqqhf.onion          port  18333    score     13
```

I'm not sure if this adds that much in its current form. Someone, who didn't look at the code, wouldn't know that `ipv4 ipv6 onion` is dynamically constructed from `networkinfo` and not just a constant. Additionally, it's for 1) quite hidden and 2) not self-explanatory that these are the reachable networks. 

Something like the following might be better:
```
reachable networks: ipv4 ipv6 onion
local addresses:
2002:b10c:b10c:0:1234::5057     port  18333    score      1
2002:b10c:b10c:0:1234::33ed     port  18333    score      1
b10cabcdefaqqqhf.onion          port  18333    score     13
```

Reply:
Thanks for the testing and feedback, @0xB10C. I'm not sure it would add much either. Dropped the commit to not invalidate your previous ACK. Thanks!